datastore/gcloud/aio/datastore/datastore.py

```diff
@@ -346,3 +346,51 @@ def make_mutation(operation: Operation, key: Key,
         transaction = await self.beginTransaction(session=session)
         mutation = self.make_mutation(operation, key, properties=properties)
         await self.commit([mutation], transaction=transaction, session=session)
+
+    async def export(self, output_bucket_name: str,
+                     kinds: Optional[List[str]] = None, namespaces: Optional[List[str]] = None,
+                     labels: Optional[Dict[str, str]]= None,
+                     session: aiohttp.ClientSession = None, timeout: int = 10) -> Dict[str, str]:
+        project = await self.project()
+        url = f'{API_ROOT}/projects/{project}:export'
+
+        export_request = {
+            'entityFilter': {'kinds': kinds or [], 'namespaceIds': namespaces or []},
+            'outputUrlPrefix': f'gs://{output_bucket_name}'}
+        if labels:
+            export_request['labels'] = labels
+
+        payload = json.dumps(export_request).encode('utf-8')
+
+        headers = await self.headers()
+        headers.update({
+            'Content-Length': str(len(payload)),
+            'Content-Type': 'application/json',
+        })
+
+        if not self.session:
+            self.session = aiohttp.ClientSession(conn_timeout=10,
+                                                 read_timeout=10)
+        session = session or self.session
+        resp = await session.post(url, data=payload, headers=headers, timeout=timeout)
+        resp.raise_for_status()
+
+        return await resp.json()
```

Comment:
Looks like a bunch of parsing on the user's side to handle the response of this. Should we create an [`Operation` class](https://cloud.google.com/datastore/docs/reference/admin/rest/Shared.Types/Operation) the same way as we have Key, Entity, etc?

Reply:
added DatastoreOperation class